### PR TITLE
[Tizen] Update default api-version of generate_sysroot script to 6.0

### PR DESCRIPTION
**Pull request by JSUYA**
sync https://github.com/flutter-tizen/embedder/pull/66

### CODE REVIEW

File: ci/tizen/generate_sysroot.py [hunk 1]
```diff
@@ -18,6 +18,8 @@
     'glibc',
     'glibc-devel',
     'libgcc',
+    'libstdc++',
+    'libstdc++-devel',
```
Comment (swift-kim) [inline ci/tizen/generate_sysroot.py]: Is this dependency necessary? Currently the engine doesn't depend on libstdc++.

```sh
$ readelf -d libflutter_engine.so

Dynamic section at offset 0x7d2adc contains 33 entries:
  Tag        Type                         Name/Value
 0x00000001 (NEEDED)                     Shared library: [libdl.so.2]
 0x00000001 (NEEDED)                     Shared library: [libfontconfig.so.1]
 0x00000001 (NEEDED)                     Shared library: [libpthread.so.0]
 0x00000001 (NEEDED)                     Shared library: [libm.so.6]
 0x00000001 (NEEDED)                     Shared library: [libgcc_s.so.1]
 0x00000001 (NEEDED)                     Shared library: [libc.so.6]
 0x00000001 (NEEDED)                     Shared library: [ld-linux.so.3]
 0x0000000e (SONAME)                     Library soname: [libflutter_engine.so]
 0x0000000c (INIT)                       0x52090
 0x0000000d (FINI)                       0x6965e4
```

Comment (JSUYA) [reply ci/tizen/generate_sysroot.py]: When I build after changing to 6.0, the error below occurs.
```
/usr/bin/arm-linux-gnueabi-ld: cannot find -lstdc++
```
Have you tried after deleting the existing sysroot?

Comment (swift-kim) [reply ci/tizen/generate_sysroot.py]: Okay but still not sure why it is necessary.

Comment (swift-kim) [inline ci/tizen/generate_sysroot.py]: The devel package should not be necessary.

Comment (JSUYA) [reply ci/tizen/generate_sysroot.py]: I removed devel package

**Review by swift-kim — APPROVED** [review]

**Pull request merged**